RedHatInsights/drift-frontend
# [DRFT-983] Change comparison filter to dropdown filter selector

Before, the comparison table filter had a fact name search filter and a state dropdown filter that were static in the toolbar. This PR adds a dropdown to select the type of filter you'd like to use (Fact name or state) and switches between the previously mentioned search filter and state dropdown filter depending on your dropdown selection.

To test:
- Go to drift app
- Click "Add to comparison" button (ensure you have at least one system or baseline)
- Select at least one system or baseline to create a comparison and click "Submit"

Test the filters and chips to ensure that filters are properly added and removed. Don't forget to test the feature when filtering by fact name that allows you to type a fact name filter and press enter. This allows to effectively set a filter and type in additional fact name filters. This filter acts as an OR filter. Filtering with "dog" and "cat" will give you facts that contain "dog" OR "cat".

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
